### PR TITLE
UI: Styling for Published Page

### PR DESCRIPTION
**Pull request by jordanvidrine**
Styling to look more visually connected to Discourse

## Before
<kbd>image</kbd>

## After
<kbd>image</kbd>

### CODE REVIEW

File: app/controllers/published_pages_controller.rb [hunk 1]
```diff
@@ -31,6 +31,9 @@ def show
 
     @topic = pp.topic
     @canonical_url = @topic.url
```
Comment (jjaffeux) [inline app/controllers/published_pages_controller.rb:33]: You might want to reuse the logic we have in application_helper:

```
  def application_logo_url
    @application_logo_url ||= begin
      if mobile_view? && SiteSetting.site_mobile_logo_url.present?
        SiteSetting.site_mobile_logo_url
      else
        SiteSetting.site_logo_url
      end
    end
  end
```

File: app/controllers/published_pages_controller.rb [hunk 2]
```diff
@@ -31,6 +31,9 @@ def show
...
+    @logo = SiteSetting.logo_small
```
Comment (jjaffeux) [inline app/controllers/published_pages_controller.rb:34]: You might want to reuse the logic we have in application_helper:

```
  def application_logo_url
    @application_logo_url ||= begin
      if mobile_view? && SiteSetting.site_mobile_logo_url.present?
        SiteSetting.site_mobile_logo_url
      else
        SiteSetting.site_logo_url
      end
    end
  end
```

File: app/views/published_pages/show.html.erb [hunk 3]
```diff
@@ -1,18 +1,24 @@
...
+    </div>
+  </div>
+<div class="published-page-content-wrapper">
+  <div class="published-page-content-header">
     <div class="published-page-author">
-      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" class="avatar">
+      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" style="border: 4px solid <%= @border_color %>90" class="avatar">
       <div class="published-page-author-details">
-        <div class="username"><%= @topic.user.username %></div>
+        <div class="username"><%= @topic.user.name %></div>
```
Comment (jjaffeux) [inline app/views/published_pages/show.html.erb:14]: do we always have a name ? I'm unsure

File: app/assets/stylesheets/publish.scss [hunk 4]
```diff
@@ -1,17 +1,47 @@
...
+.published-page-content-wrapper {
   margin: 2em auto;
   max-width: 800px;
 }
 
+.pp-header {
+  width: 100%;
+  top: 0;
+  z-index: 1000;
+  background-color: #fff;
```
Comment (jjaffeux) [inline app/assets/stylesheets/publish.scss:12]: should be $secondary no ?

File: app/assets/stylesheets/publish.scss [hunk 5]
```diff
@@ -1,17 +1,47 @@
...
-.published-page-wrapper {
+.published-page-content-wrapper {
   margin: 2em auto;
   max-width: 800px;
 }
 
+.pp-header {
+  width: 100%;
+  top: 0;
+  z-index: 1000;
```
Comment (jjaffeux) [inline app/assets/stylesheets/publish.scss:11]: should maybe use the scss helper for z-index ?

File: app/controllers/published_pages_controller.rb (hunk 2 above)
Comment (jordanvidrine) [reply app/controllers/published_pages_controller.rb:34]: I did try this out, but it returns the full width site logo on desktop. I want the small/mobile logo to appear here regardless of desktop/mobile user.

<kbd>
image</kbd>

File: app/views/published_pages/show.html.erb [hunk 6]
```diff
@@ -1,18 +1,24 @@
...
+        <img class="pp-logo" src="<%=@logo.url%>"/>
+      </a>
+      <h1 class="pp-title"><%= @topic.title %></h1>
+    </div>
+  </div>
+<div class="published-page-content-wrapper">
+  <div class="published-page-content-header">
     <div class="published-page-author">
-      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" class="avatar">
+      <img width="45" height="45" src="<%= @topic.user.avatar_template.gsub('{size}', '90') %>" style="border: 4px solid <%= @border_color %>90" class="avatar">
```
Comment (jjaffeux) [inline app/views/published_pages/show.html.erb:12]: as I hate inline style, something we could at least do here is only having border-color :p and move width/style in css

**Pull request merged**